client/src/components/layout/ReviewDeleteButton.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+const ReviewDeleteButton = ({ user, review, getPlanet }) => {
+  
+  const deleteReview = async (event) => {
+    event.preventDefault()
+    const reviewBody = {
+      reviewId: review.id,
+    }
+    try {
+      const response = await fetch(`/api/v1/planets/${review.planetId}/reviews/delete-review`, {
```

Comment:
interestingly, you dont need a nestedRouted for this. I know its confusing.
This should post to a `reviewsRouter.delete(/:id)` route. It may be require a new reviewsRouter

Reason being is that you dont need the Planet to delete the review, just its ID. You will not need to send a `body` prob along. The id will be in URL params and you can use that to delete your review on the backend